crates/autopilot/src/run_loop.rs

```diff
@@ -513,6 +517,28 @@ pub fn solve_request(
                             .collect()
                     };
                 let order_is_untouched = remaining_order.executed_amount.is_zero();
+                let fee_policy = match (
+                    fee_policy.price_improvement_factor,
+                    fee_policy.volume_factor,
```

Comment:
Let's not semantically overload this config parameter please. No one will guess that volume factor means different things depending on whether or not another parameter is set.

Reply:
After refactoring the argument to enum, now the conversion is 1 to 1